lib/boxxspring/worker/metrics.rb

```diff
@@ -22,36 +22,38 @@ def initialize( *arguments )
       end
 
       def upload_metrics
-        loop do 
+        loop do
           unless @metrics.empty?
             begin
-              metrics_payload = nil 
-              
+              metrics_payload = nil
+
               METRICS_MUTEX.synchronize do
-                metrics_payload = @metrics
-                @metrics = []
+                logger.info( "Metrics queue has #{@metrics.count} metrics" ) \
```

Comment:
Can you change this to an `if` block instead of moving the condition to the next line